x-pack/plugins/lens/kibana.json

```diff
@@ -13,7 +13,7 @@
     "dashboard",
     "charts"
   ],
-  "optionalPlugins": ["embeddable", "usageCollection", "taskManager", "uiActions"],
+  "optionalPlugins": ["usageCollection", "taskManager", "uiActions", "embeddable"],
```

Comment:
Once `attributeService` is moved back into embeddable, it will no longer be optional due to the usage of `attributeService` in `app.tsx`. For now, it is still optional.